test/with_dummyserver/test_connectionpool.py

```diff
@@ -231,7 +231,7 @@ def test_upload(self) -> None:
             "upload_filename": "lolcat.txt",
             "filefield": ("lolcat.txt", data),
         }
-        fields["upload_size"] = len(data)  # type: ignore
+        fields["upload_size"] = len(data)  # type: ignore[assignment]
```

Comment:
I didn't check if such a change to this line affects test coverage, but it looks to fix the ignored mypy error

```suggestion
        fields["upload_size"] = str(len(data))
```

Reply:
Sorry: but I don't care to change this